client/landing/stepper/declarative-flow/internals/steps-repository/processing-step/hooks/use-processing-loading-messages.tsx

```diff
@@ -153,6 +156,26 @@ export function useProcessingLoadingMessages( flow?: string | null ): LoadingMes
 					},
 				];
 		}
+	} else if ( VIDEOPRESS_FLOW === flow ) {
+		const videoPressLoadingMessages = [
+			{ title: __( 'Setting up your video site' ), duration: 5000 },
+			{ title: __( 'Scouting the locations' ), duration: 5000 },
+			{ title: __( 'Kicking off the casting' ), duration: 5000 },
+			{ title: __( "Let's head to the checkout" ), duration: 5000 },
+		];
+		return videoPressLoadingMessages;
+	} else if ( VIDEOPRESS_TV_FLOW === flow ) {
+		const videoPressLoadingMessages = [
+			{ title: __( 'Starting up your channel' ), duration: 5000 },
+		];
+		return videoPressLoadingMessages;
+	} else if ( VIDEOPRESS_TV_PURCHASE_FLOW === flow ) {
+		const videoPressLoadingMessages = [
+			{ title: __( 'Scouting the locations' ), duration: 5000 },
+			{ title: __( 'Kicking off the casting' ), duration: 5000 },
+			{ title: __( "Let's head to the checkout" ), duration: 5000 },
```

Comment:
This `'` should maybe be one of those fancy apostrophes?
```suggestion
			{ title: __( 'Let’s head to the checkout' ), duration: 5000 },
```